NotExistsShapeException.java

```diff
@@ -0,0 +1,15 @@
+
+public class NotExistsShapeException extends Exception {
+	
+    // Parameterless Constructor
+      public NotExistsShapeException() {
+  
+    }
+
+    // Constructor that accepts a message
+    public  NotExistsShapeException(String message)
+    {
+       super(message);
+    }

```

Comment:
If you decided to make an exception class, you should override all 4 constructors. It is essential that a ctor which takes another exception exists for exception chaining.